examples/Server/Interceptors/LoggingInterceptor.cs

```diff
@@ -0,0 +1,52 @@
+﻿
+using System.Threading.Tasks;
+using Grpc.Core;
+using Grpc.Core.Interceptors;
+using Microsoft.Extensions.Logging;
+
+namespace Server.Interceptors
+{
+    public class LoggingInterceptor : Interceptor
+    {
+        ILogger _logger;
+        string _prefix;
+
+        public LoggingInterceptor(ILogger<LoggingInterceptor> logger, string prefix)
```

Comment:
Is the prefix to test passing values via AddInterceptor? How about changing prefix to environment.

Log message can then be something like: `Beginning request to {context.Method} on {environment}`

Reply:
More for testing the Add<TInterceptor>(params object[] args) method. If it were just environment, it would be resolved from DI not the args parameter. I agree that the example is very contrived but again, I don't think this example will remain.